### ISSUE TITLE
Xbox controller (One, Series X|S) doesn't work properly

### ISSUE THREAD

**Issue by ArieKindicator** [ppeccin/WebMSX#68]
I have some issues with a Xbox controller (Series X|S and Xbox One).
When I launch WebMSX in the MS Edge browser on Xbox Series X the controller is automatically detected but the only buttons that are responsive [in "Use game control" mode] are LB and LT = Slow motion, RB and RT = fast forward, View = toggle full screen on/off, Menu = toggle "Use browsing controls" [default navigation controls in MS Edge on Xbox] / "Use game controls" other buttons [A, B] don't respond (visibly) to user input.

When I launch WebMSX on webmsx.org I'm able to toggle between controller/game control and browsing modes when the cursor is positioned outside the active WebMSX 'frame'. That is when the cursor is in the white area on the screen. Inside the WebMSX frame this isn't possible. And that is a bit problematic.
To confirm the opening message "For the best experience … NICE!" I have to switch to "browsing" mode [default Xbox navigation] because otherwise you can't accept or pass this message (the A and B buttons do't respond in game mode).

At the launch of WebMSX I'm able to switch to browsing mode and I can load a game (and change other WebMSX settings) but to play a game with the controller you have to switch back to controller/game control mode. This isn't possible because somehow, after the acceptance of the opening message the whole screen becomes the active WebMSX 'frame' and to toggle between game controls and browsing controls can only be done outside this WebMSX frame. Note: toggle full screen on/off [View button] and / or turning touch controls off doesn't solve this. The whole screen/page including the black background or border remains the active WebMSX frame. 

I don't have a solution, but is it possible to enable "joystick control" to navigate / change the WebMSX menu and settings. This should only require the D pad and A or B button for confirmation. Other inputs can be done with the virtual keyboard.

**Comment by ppeccin** [comment]
Hi!

Well, I could not understand very well what is working and what is not,
especially from your first paragraph.
Can you try explaining this in simpler terms.

I don't one any XBox device so I cannot test this.
Start by letting me know how the controls work on the XBox browser, what
buttons your controller has, their functions on the browser, etc, since I
don't know anything about it!

Regards,
Paulo
